test/hashing.jl

```diff
@@ -249,7 +250,9 @@ end
         )
 
         for a in vals, b in vals
-            @test isequal(a, b) == (Base.hash_64_32(a) == Base.hash_64_32(b))
+            ha = Base.hash_64_32(a, Base.HASH_SEED, Base.HASH_SECRET)
```

Comment:
I don't think these 3-argument methods exist?

Reply:
I think you're right, but I swear tests were passing somehow...